CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   When upgrading from Strimzi 0.22 or earlier, you have to disable the `ControlPlaneListener` feature gate when upgrading the cluster operator to make sure the Kafka cluster stays available during the upgrade.
   When downgrading to Strimzi 0.22 or earlier, you have to disable the `ControlPlaneListener` feature gate before downgrading the cluster operator to make sure the Kafka cluster stays available during the downgrade.
 * Update to Cruise Control version 2.5.74
+* The `type` field in `ListenerStatus` have been deprecated and will be removed when updating to CRD v1
```

Comment:
```suggestion
* The `type` field in `ListenerStatus` has been deprecated and will be removed when updating to CRD v1
```

Reply:
The comment from Tom here still stands I guess. plus you need to more this to the 0.28.0 release.